packages/access-api/src/routes/validate-email.js

```diff
@@ -134,6 +138,46 @@ async function session(req, env) {
     req.query.ucan,
     delegation.capabilities[0].nb.key
   )
+  if (req.method.toLowerCase() === 'post') {
+    const accessSessionResult = await validator.access(delegation, {
+      capability: Access.session,
+      principal: Verifier,
+      authority: env.signer,
+    })
+    if (accessSessionResult.error) {
+      throw new Error(
+        `unable to validate access session: ${accessSessionResult.error}`
+      )
+    }
+    const account = { did: () => accessSessionResult.audience.did() }
```

Comment:
nit: You don't need to wrap you could just

```suggestion
    const account = accessSessionResult.audience
```